trite-osprey/src/App.css

```diff
@@ -0,0 +1,156 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  animation: App-logo-spin infinite 10s linear;
+  height: 80px;
+  filter: invert(100%);
+}
+
+.App-header {
+  background-color: #222;
+  height: 150px;
+  padding: 20px;
+  color: white;
+}
+
+.App-intro {
+  font-size: large;
+  padding: 1em;
+}
+h2{
+  font-family: 'Permanent Marker', cursive;
+  font-size: 3em;
+
+}
+@keyframes App-logo-spin {
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+}
+
+.listclass {
+  display: flex;
+  align-items: center;
+  z-index: 1;
+}
+
+input[type = text]{
```

Comment:
You want to avoid styles like this. Styling `input[type = text]` locks you into those styles for all text inputs. Trust me you don't want that kind of inflexibility. Make up a classname. `.awesome-input` or something. Anything :)